clang/lib/CodeGen/CGCall.cpp

```diff
@@ -5520,6 +5520,12 @@ RValue CodeGenFunction::EmitCall(const CGFunctionInfo &CallInfo,
       CGM.AdjustMemoryAttribute(CalleePtr->getName(), Callee.getAbstractInfo(),
                                 Attrs);
   }
+  // We may not have a FunctionDecl*, but we still need to support strictfp.
```

Comment:
This comment is explaining the patch, not explaining the code as it stands after the patch. I suggest removing this and just pulling the comment inside the `if` up here, since it explains why this block has to exist.